### PR TITLE
First pass at MODIS Riverscapes Type

### PR DESCRIPTION
**Pull request by MattReimer**
@philipbaileynar please take a look at this. If there are any naming or metadata changes now is the time.

### CODE REVIEW

File: Programs/BRAT_CRB/Program.xml [hunk 1]
```diff
@@ -32,6 +36,7 @@
       <Product id="BRAT" name="BRAT Models (HUC8)" folder="BRAT" url="" />
       <Product id="VBET" name="Valley Bottom Extraction Tool (VBET)" folder="VBET" url="" />
       <Product id="RSContext" name="Riverscapes Context (HUC8)" folder="RSContext" url="" />
+      <Product id="MODIS" name="MODIS (HUC4)" folder="MODIS" url="" />
```
Comment (MattReimer) [inline Programs/BRAT_CRB/Program.xml:39]: This `name` is what it will look like in the dropdown

File: Projects/XSD/V1/RiverscapesProject.xsd [hunk 2]
```diff
@@ -137,6 +137,7 @@
       <xs:element type="SimpleFileType" name="SQLiteDB" />
       <xs:element type="SimpleFileType" name="PDF" />
       <xs:element type="SimpleFileType" name="HTMLFile" />
+      <xs:element type="SimpleFileType" name="ZipFile" />
```
Comment (MattReimer) [inline Projects/XSD/V1/RiverscapesProject.xsd:140]: New file type

File: Projects/examples/MODIS.xml [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+  <Meta name="HUC4">1706</Meta>
+  <Meta name="HUC">1706</Meta>
+</MetaData>
+<Realizations>
+  <MODIS id="MODIS1" dateCreated="2020-10-27T15:08:11.598597" guid="1cba8df3-f576-432d-b784-6018220224f2" productVersion="0.0.1">
+    <Name>MODIS for HUC 1706</Name>
+    <Outputs>
+      <ZipFile>
+        <Name>MODIS Results</Name>
+        <Path>1706.zip</Path>
```
Comment (MattReimer) [inline Projects/examples/MODIS.xml:18]: Any metadata?

File: Projects/examples/MODIS.xml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+<?xml version='1.0' encoding='utf-8'?>
+<Project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" 
+xsi:noNamespaceSchemaLocation="../XSD/V1/MODIS.xsd">
+<Name>MODIS for HUC 1706</Name>
+<ProjectType>MODIS</ProjectType>
+<MetaData>
+  <Meta name="ModelVersion">0.0.1</Meta>
+  <Meta name="dateCreated">2020-10-28T15:08:11.579051</Meta>
+  <Meta name="HUC4">1706</Meta>
+  <Meta name="HUC">1706</Meta>
```
Comment (MattReimer) [inline Projects/examples/MODIS.xml:10]: Extra metadata?

File: Programs/BRAT_CRB/Program.xml (hunk 1 above)
Comment (philipbaileynar) [reply Programs/BRAT_CRB/Program.xml:39]: Sorry, I know we refer to this as MODIS within NAR, but can we use the name **LST** or Land Surface Temperature instead.

Someone could conceivably generate the same data from a different sensor or method.

Also, I was going to request that we abandon the "HUCx" suffixes in the product names. Can leave the legacy ones for now. But let's not add more products with these labels.

File: Projects/XSD/V1/RiverscapesProject.xsd (hunk 2 above)
Comment (philipbaileynar) [reply Projects/XSD/V1/RiverscapesProject.xsd:140]: I like this. Could be useful further down the road for either bundling new file types without the need to constantly change the XSD, or incorporating big files (as we are doing here).

File: Projects/examples/MODIS.xml (hunk 3 above)
Comment (philipbaileynar) [reply Projects/examples/MODIS.xml:18]: Replace all references to MODIS with LST

**Pull request merged**